application/src/main/java/org/thingsboard/mqtt/broker/install/ThingsboardMqttBrokerInstallService.java

```diff
@@ -36,64 +37,40 @@ public class ThingsboardMqttBrokerInstallService {
-    @Value("${install.upgrade.from_version:1.2.3}")
-    private String upgradeFromVersion;
 
     private final DatabaseSchemaService databaseSchemaService;
     private final ApplicationContext context;
     private final SystemDataLoaderService systemDataLoaderService;
     private final DatabaseEntitiesUpgradeService databaseEntitiesUpgradeService;
     private final CacheCleanupService cacheCleanupService;
     private final DataUpdateService dataUpdateService;
+    private final DatabaseSchemaSettingsService databaseSchemaVersionService;
 
     public void performInstall() {
         try {
             if (isUpgrade) {
-                log.info("Starting TBMQ Upgrade from version {} ...", upgradeFromVersion);
-
-                cacheCleanupService.clearCache(upgradeFromVersion);
-
-                switch (upgradeFromVersion) {
-                    case "1.0.0":
-                    case "1.0.1":
-                        log.info("Upgrading TBMQ from version 1.0.1 to 1.1.0 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("1.0.1");
-                    case "1.1.0":
-                        log.info("Upgrading TBMQ from version 1.1.0 to 1.2.0 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("1.1.0");
-                    case "1.2.0":
-                        log.info("Upgrading TBMQ from version 1.2.0 to 1.2.1 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("1.2.0");
-                    case "1.2.1":
-                        log.info("Upgrading TBMQ from version 1.2.1 to 1.3.0 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("1.2.1");
-
-                        systemDataLoaderService.createWebSocketMqttClientCredentials();
-                        systemDataLoaderService.createDefaultWebSocketConnections();
-                    case "1.3.0":
-                        log.info("Upgrading TBMQ from version 1.3.0 to 1.4.0 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("1.3.0");
-                        dataUpdateService.updateData("1.3.0");
-                    case "1.4.0":
-                        log.info("Upgrading TBMQ from version 1.4.0 to 2.0.0 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("1.4.0");
-                        dataUpdateService.updateData("1.4.0");
-                    case "2.0.0":
-                        log.info("Upgrading TBMQ from version 2.0.0 to 2.0.1 ...");
-                        databaseEntitiesUpgradeService.upgradeDatabase("2.0.0");
-                        dataUpdateService.updateData("2.0.0");
-                        break;
-                    default:
-                        throw new RuntimeException("Unable to upgrade TBMQ, unsupported fromVersion: " + upgradeFromVersion);
-                }
-
+                // TODO DON'T FORGET to update SUPPORTED_VERSIONS_FROM in DefaultDatabaseSchemaSettingsService
```

Comment:
correct comment would be 

// TODO DON'T FORGET to update SUPPORTED_VERSIONS_FOR_UPGRADE in DefaultDatabaseSchemaSettingsService